src/pages/Facility/services/inventory/SupplyDeliveryTable.tsx

```diff
@@ -38,6 +38,52 @@ import { ShortcutBadge } from "@/Utils/keyboardShortcutComponents";
 import mutate from "@/Utils/request/mutate";
 import { EllipsisVertical } from "lucide-react";
 
+function calculateLineTotal(
```

Comment:
same logic as used in DeliveryOrderShow, we can extract it out if we are keeping it on FE side.